app/models/api_connection.rb

```diff
@@ -4,6 +4,7 @@
 #
 #  id                    :bigint           not null, primary key
 #  api_token_private_key :string           not null
+#  name                  :string
```

Comment:
navrhoval by som tu to nazvat `custom_name` aby si nemusel carovat s tym `super` nizsie, lebo to je pomerne nestandardne.